src/gui/editors/PianoRoll.cpp

```diff
@@ -1560,6 +1562,8 @@ void PianoRoll::keyReleaseEvent(QKeyEvent* ke )
 				update();
 			}
 			break;
+		default:
+			ke->ignore();
```

Comment:
I would add a `break` after this line. Its safer IMO if we don't intend on doing fall-through at all, even if its the last case evaluated.